ecosystem/sep-0024.md

```diff
@@ -86,7 +86,7 @@ SEP-24 lays out many options for how deposit and withdrawal can work. These are
 * **Make a request to `/deposit` or `/withdraw`.**
     * This will respond with the interactive url needed to proceed with KYC and deposit/withdraw details.
 * **For `/deposit` and `/withdraw`**
-    * Optionally attach any fields from [SEP-9](sep-0009.md) as query parameters in the interactive URL, in order to let the anchor pre-fill them in the interactive flow UI.  This is optional, but can create a much nicer user experience.
+    * Optionally attach any fields from [SEP-9](sep-0009.md) as query parameters in the interactive URL, in order to let the anchor pre-fill them in the interactive flow UI.  This is optional, but can create a much nicer user experience.  Email address and name are good examples of fields to help pre-fill the anchors website.
```

Comment:
I think this should be `email_address` instead of "Email address", ditto for `name`, to hammer the point home that these are specifically-named fields.

Reply:
Good call